modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/systemDataSets.spec.ts

```diff
@@ -90,125 +103,253 @@ test(
+			await expect(form.urlInput).toHaveValue('#');
+			await expect(form.headlessActionKeyInput).toHaveValue('view');
+			await expect(form.confirmationMessageInput).toHaveValue(
+				'Are you sure?'
+			);
+			await expect(form.confirmationMessageTypeSelect).toHaveValue(
+				'danger'
+			);
+
+			await form.cancelButton.click();
+		});
+
+		await test.step('Creation actions are imported with "detached" import policy', async () => {
+			await actionsPage.selectTab({
+				container: actionsPage.actionsTabs,
+				label: 'Creation Actions',
 			});
 
-			await test.step('Check item actions are imported', async () => {
-				await actionsPage.open({dataSetLabel: 'Customized Sample'});
+			const creationActionRow = actionsPage.creationActionsTable
+				.locator('tr')
+				.filter({hasText: 'Open Form'})
+				.first();
 
-				const itemActionRow = actionsPage.itemActionsTable
-					.locator('tr')
-					.filter({hasText: 'Nav Links'})
-					.first();
+			await creationActionRow.locator('.dropdown-toggle').click();
 
-				await itemActionRow.locator('.dropdown-toggle').click();
+			await actionsPage.page
+				.locator('.dropdown-menu.show')
+				.getByRole('menuitem', {name: 'Edit'})
+				.click();
+
+			const form = actionsPage.actionForm;
+
+			await expect(form.labelInput).toHaveValue('Open Form');
+			await expect(form.iconInput).toHaveValue('bolt');
+			await expect(form.typeSelect).toHaveValue('modal');
+			await expect(form.variantSelect).toHaveValue('full-screen');
+			await expect(form.titleInput).toHaveValue('My Products');
+			await expect(form.urlInput).toHaveValue('#');
+			await expect(form.headlessActionKeyInput).toHaveValue('update');
+
+			await form.cancelButton.click();
+
+			await page.getByTitle('Back').click();
+		});
 
-				await actionsPage.page
+		await test.step('Item actions are imported with "item proxy" import policy', async () => {
+			await actionsPage.open({dataSetLabel: 'Classic Sample'});
+
+			const itemActionRows = actionsPage.itemActionsTable
+				.locator('tr')
+				.filter({hasText: 'ITEM_PROXY'});
+
+			await expect(itemActionRows).toHaveCount(2);
+
+			const firstDropdownToggle = itemActionRows
+				.first()
+				.locator('.dropdown-toggle');
+
+			await firstDropdownToggle.click();
+
+			await expect(
+				actionsPage.page
 					.locator('.dropdown-menu.show')
 					.getByRole('menuitem', {name: 'Edit'})
-					.click();
-
-				const form = actionsPage.actionForm;
-
-				await expect(form.labelInput).toHaveValue('Nav Links');
-				await expect(form.iconInput).toHaveValue('home');
-				await expect(form.typeSelect).toHaveValue('link');
-				await expect(form.urlInput).toHaveValue('#');
-				await expect(form.headlessActionKeyInput).toHaveValue('view');
-				await expect(form.confirmationMessageInput).toHaveValue(
-					'Are you sure?'
-				);
-				await expect(form.confirmationMessageTypeSelect).toHaveValue(
-					'danger'
-				);
-
-				await form.cancelButton.click();
+			).toBeHidden();
```

Comment:
Personally, I dislike the `.toBeHidden()` not making any distinction between "DOM node not found" and "DOM node not visible". Here we want to assert the first. There seems to be no better way, so let's go with this.

Reply:
Updated in https://github.com/liferay-frontend/liferay-portal/pull/4695/commits/3cad9c90d5748db3776f0c30ccc4a3ced40a42fa